build/rofl/manifest.go

```diff
@@ -292,6 +307,27 @@ func (d *Deployment) HasAppID() bool {
 	return len(d.AppID) > 0
 }
 
+// Instance is a hosted instance where a ROLF app is deployed.
+type Instance struct {
+	// Provider is the address of the ROFL market provider to deploy to.
+	Provider string `yaml:"provider,omitempty" json:"provider,omitempty"`
+	// Offer is the provider's offer identifier to provision.
+	Offer string `yaml:"offer,omitempty" json:"offer,omitempty"`
+	// Instance is the identifier of the instance to deploy into.
+	Instance string `yaml:"instance,omitempty" json:"instance,omitempty"`
```

Comment:
Will this rented *instance* term clash with the ROFL node *instance* term? Could we call it sth else?

Reply:
Changed.